src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File fileFromFileName = new File(fromFileName);
+        File fileToFileName = new File(toFileName);
+        Map<String, Integer> result = new HashMap<>();
+
+        if (!fileFromFileName.exists() || !fileToFileName.exists()) {
+            return;
+        }
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String lineData;
+
+            while ((lineData = bufferedReader.readLine()) != null) {
+                String[] parts = lineData.split(" ");
+                for (String part : parts) {
```

Comment:
The loop over 'parts' is unnecessary if each line contains only one 'word,number' pair. Remove the loop to simplify the code.